aspnet/web-forms/overview/getting-started/index.md

```diff
@@ -33,3 +33,5 @@ msc.type: chapter
     - [What's New in ASP.NET and Web Development in Visual Studio 2012](hands-on-labs/whats-new-in-aspnet-and-web-development-in-visual-studio-2012.md)
     - [What's New in Web Forms in ASP.NET 4.5](hands-on-labs/whats-new-in-web-forms-in-aspnet-45.md)
     - [Using Page Inspector in Visual Studio 2012](hands-on-labs/using-page-inspector-in-visual-studio-2012.md)
+
+[!INCLUDE[](~/includes/webLivePreview.md)]
```

Comment:
@bzhu94 copy/paste this line to have the content added to any file.

Reply:
To add this to any article, At the top right of the article, select the Edit pen icon:

![image](https://user-images.githubusercontent.com/3605364/53368931-11972200-38ee-11e9-86ce-18aaff69c719.png)

Select the pen icon again.  Edit, then Save.